console-csharp-connect-sample/GraphClient.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using System.Linq;
+using console_csharp_connect_sample.Helpers;
+
+namespace console_csharp_connect_sample
+{
+	// This class holds the graph client data and 
+	// returns a fully constructed instance of the 
+	// GraphServiceClient with the client data to be 
+	// used when authenticating requests to the Graph API
+	public class GraphClient
+	{
+		private GraphServiceClient ServiceClient;
+		private string ClientId { get; set; }
+		private string Authority { get; set; }
+		private string[] Scopes { get; set; }
```

Comment:
Same feedback as for the other sample IEnumerable&lg;string&gt; ?
Also you might want to have a public set? so that the developer sets the Scopes before calling an API?